docs/public/using-devfile-odo.dev.push.path-attribute.adoc

```diff
@@ -0,0 +1,29 @@
+# Using the odo.dev.push.path related attributes
+
+`odo` uses the `odo.dev.push.path` related attribute from the devfile's run commands to push only specific files and folders to the component. The format of the attribute is `odo.dev.push.path:<local_relative_path>:<remote_relative_path>`
+
+```yaml
+commands:
+  - id: dev-run
+    attributes:
+      "dev.odo.push.path:target/quarkus-app": "target/quarkus-app"
```

Comment:
Could we have different local and remote paths, for the reader to be sure to understand what is local and what is remote in the other parts of the file?